lib/pbench/agent/base.py

```diff
@@ -0,0 +1,97 @@
+import abc
+import datetime
+import os
+import socket
+import sys
+from pathlib import Path
+
+import click
+import six
```

Comment:
Let's consider removing the use of `six` to detect Python2 environments.

It is not clear we want to embed knowledge of what we require for Python3 into the code.

Reply:
done